### DISCUSSION TITLE
native侧是如何将flutter引擎打包的？

### DISCUSSION THREAD

**Discussion by nishibaiyang** [openkraken/kraken#473] [Q&A]
可以介绍下native侧是如何执行bundle的，主要是想了解原生侧打包成aar的过程

**Comment by andycall** [comment]
打包是直接调用 flutter build apk 进行打包，后面都是 flutter 官方提供的工具链

![image](https://user-images.githubusercontent.com/4409743/124590930-476d4d80-de8e-11eb-97df-e78017aead5b.png)
![image](https://user-images.githubusercontent.com/4409743/124590977-548a3c80-de8e-11eb-8dd4-cafbd57b3ae0.png)


参考资料：
+ https://flutter.dev/docs/development/add-to-app

kraken 基于 flutter 命令封装的工具脚本

+ https://github.com/openkraken/kraken/blob/main/scripts/build_android_sdk.js
+ https://github.com/openkraken/kraken/blob/main/scripts/tasks.js#L472
